FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/viewmodel/MessagesViewModel.kt

```diff
@@ -535,7 +535,7 @@ class MessagesViewModel(application: Application) : AccountViewModel(application
     GmailApiHelper.identifyAttachments(msgEntities, msgs, account, localFolder, roomDatabase)
     val session = Session.getInstance(Properties())
     updateLocalContactsIfNeeded(messages = msgs
-      .filter { it.labelIds.contains(GmailApiHelper.LABEL_SENT) }
+      .filter { it.labelIds?.contains(GmailApiHelper.LABEL_SENT) == true }
```

Comment:
From my point of view, the use of the `Elvis` operator to handle null values is a best practice in Kotlin. It improves the readability and clarity of the code by handling null values in a concise and explicit way.

I think above approach is appropriate because it ensures that we filter messages only when the `labelIds` list is not null and contains the `LABEL_SENT` label.

However, the preferred approach may vary depending on the developer's point of view and you may choose your preferred way.

Reply:
Yes

Reply:
As you may know, I also do other programming languages (swift, javascript, etc). It is not customary for me to compare the returned value of a function that already returns a Boolean (such as `isNotEmpty` in this case) to the Boolean value of true.

Reply:
I may be overthinking this. If you are already familiar with your preferred method, please feel free to continue using it :-)

Reply:
> I may be overthinking this. If you are already familiar with your preferred method, please feel free to continue using it :-)

No-no, Please always share your thoughts. It's very important for our experience and improvements. Most of your comments were great and helped me improve my code. Usually, you find what I overlooked )). Thank you for that!

Reply:
I see. If IDE recommends that way, let's proceed with that way.